systemvm/scripts/config_ssl.sh

```diff
@@ -120,7 +120,7 @@ customCACert=
 publicIp=
 hostName=
 keyStore=$(dirname $0)/certs/realhostip.keystore
-defaultJavaKeyStoreFile=/etc/ssl/certs/java/cacerts
+defaultJavaKeyStoreFile="$(jrunscript -e 'java.lang.System.out.println(java.lang.System.getProperty("java.home"));')/lib/security/cacerts"
```

Comment:
LGTM for 4.10, however for 4.11 we're going back to openjdk-8 provided in Debian "Stretch" where this may not be needed. Maybe you want to send this PR for 4.10?

Reply:
Hi @rhtyd 
yes we can do it as fix for 4.10. no problem. is it clear that 4.11 will use the old path and for longer time or do we need a more ultimate fix?

Sven

Reply:
how we can change that to 4.10?